src/base/Dockerfile.in

```diff
@@ -3,200 +3,43 @@
-    texlive \
-    texinfo \
-    texlive-fonts-extra \
-    texlive-latex-extra
-<% end %>
-
-#<% if !use_r_devel %>
-
-RUN apt-get update && \
-    apt-get -y install --fix-missing --fix-broken \
-    texlive \
-    texinfo \
-    texlive-fonts-extra \
-    texlive-latex-extra
-
-#<% end %>
-
-
-<% end %>
-
-<% if use_r_devel %>
-# There should only be one version of R on this container
-#RUN dpkg --purge --force-depends r-base-core
-# FIXME figure out how to REALLY remove r-base-core and deps
-# so that it does not come back.
-
-<% if use_numbered_branch.is_a? String %>
-RUN rm -rf /usr/lib/R/library /usr/bin/R /usr/bin/Rscript
-
-RUN apt-get remove -y libfreetype6; echo "ignoring return value"
-
-
-RUN cd /tmp && \
-svn co https://svn.r-project.org/R/branches/R-<%= use_numbered_branch.sub(".", "-")%>-branch R \
-  && cd R \
-  && DEBIAN_FRONTEND=noninteractive apt-get install -y \
-		libblas-dev \
-		libbz2-dev  \
-		libcairo2-dev \
-		libfontconfig1-dev \
-		libfreetype6-dev \
-		libglib2.0-dev \
-		libjpeg-dev \
-		liblapack-dev  \
-		liblzma-dev \
-		libncurses5-dev \
-		libpango1.0-dev \
-		libpcre3-dev \
-		libpng12-dev \
-		libreadline-dev \
-		libtiff5-dev \
-		libxft-dev \
-		libxt-dev \
-		r-base-dev \
-		tcl8.6-dev \
-		texlive-base \
-		texlive-fonts-recommended \
-		texlive-generic-recommended \
-		texlive-latex-base \
-		texlive-latex-recommended \
-		tk8.6-dev && \
-    R_PAPERSIZE=letter \
-		R_BATCHSAVE="--no-save --no-restore" \
-		R_BROWSER=xdg-open \
-		PAGER=/usr/bin/pager \
-		PERL=/usr/bin/perl \
-		R_UNZIPCMD=/usr/bin/unzip \
-		R_ZIPCMD=/usr/bin/zip \
-		R_PRINTCMD=/usr/bin/lpr \
-		LIBnn=lib \
-		AWK=/usr/bin/awk \
-		CFLAGS=$(R CMD config CFLAGS) \
-		CXXFLAGS=$(R CMD config CXXFLAGS) \
-		./configure --prefix=/usr/local --enable-R-shlib \
-			--without-blas \
-			--without-lapack \
-			--with-readline \
-			--without-recommended-packages \
-			--program-suffix=dev && \
-		make  && \
-		make install && \
-	  cd / && \
-	  rm -rf /tmp/R /tmp/downloaded_packages/ /tmp/*.rds \
-  && echo "R_LIBS=\${R_LIBS-'/usr/local/lib/R/site-library:/usr/local/lib/R/library:/usr/lib/R/library'}" >> /usr/local/lib/R/etc/Renviron \
-    && echo 'options("repos"="http://cran.rstudio.com")' >> /usr/local/lib/R/etc/Rprofile.site
-RUN dpkg --purge  \
-		libblas-dev \
-		libbz2-dev  \
-		libcairo2-dev \
-		libfontconfig1-dev \
-		libfreetype6-dev \
-		libglib2.0-dev \
-		libjpeg-dev \
-		liblapack-dev  \
-		liblzma-dev \
-		libncurses5-dev \
-		libpango1.0-dev \
-		libpcre3-dev \
-		libpng12-dev \
-		libreadline-dev \
-		libtiff5-dev \
-		libxft-dev \
-		libxt-dev \
-		r-base-dev \
-		tcl8.6-dev \
-		texlive-base \
-		texlive-fonts-recommended \
-		texlive-generic-recommended \
-		texlive-latex-base \
-		texlive-latex-recommended  ; \
-		echo "ignoring return value"
-RUN apt-get autoremove -qy
-
-
-<% end  #  use_numbered_branch %>
-
 # issues with '/var/lib/dpkg/available' not found
 # this will recreate
 RUN dpkg --clear-avail
```

Comment:
Looks like this is only done on devel but if useful would be useful on both release and devel?

Reply:
Yes, I can make that change.